File: 프로그래머스/0/181919. 콜라츠 수열 만들기/콜라츠 수열 만들기.js
function solution(n) {
    var answer = [];
    var x = n;
    
    while (true) {
        answer.push(x)
        if ( x == 1 ) {
            return answer;
            break;
        } else if ( x % 2 == 0 ) {
            x /= 2
        } else if ( x % 2 != 0 ) {
            x = 3 * x + 1
        }
    }
}

File: 프로그래머스/0/181919. 콜라츠 수열 만들기/콜라츠 수열 만들기.py
def solution(n):
    answer = []
    x = n;
    
    while True:
        answer.append(x)
        if x == 1:
            return answer
            break;
        elif x % 2 == 0:
            x /= 2
        elif x % 2 != 0:
            x = 3 * x + 1